components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/listener/AbstractApplicationMgtListener.java

```diff
@@ -51,6 +51,25 @@ public boolean doPostDeleteApplication(String applicationName, String tenantDoma
         return true;
     }
 
+    public boolean doPreGetApplication(String applicationName, String tenantDomain)
+            throws IdentityApplicationManagementException{
+        return true;
+    }
+
+    public boolean doPostGetApplication(ServiceProvider serviceProvider, String applicationName, String tenantDomain) throws IdentityApplicationManagementException{
+        return true;
+    }
+
+    public boolean doPreGetApplicationByClientId(String clientId, String clientType,
+                                                 String tenantDomain)throws IdentityApplicationManagementException{
+        return true;
+    }
+
+    public boolean doPostGetApplicationByClientId(ServiceProvider serviceProvider, String clientId, String clientType,
+                                                 String tenantDomain)throws IdentityApplicationManagementException{
+        return true;
+    }
```

Comment:
In all above methods rename Application to ServiceProvider to be consistent with service methods.